frontend/src/store/modules/place/state.js

```diff
@@ -1,4 +1,5 @@
 export default {
     places: [],
-    currentPlace: null
+    liked: 'LIKED' | 'DISLIKED' | 'NONE',
```

Comment:
'LIKED' | 'DISLIKED' | 'NONE' this won't work, I wrote this for example like general notation =) Create constants 
```
const STATUS_LIKED = 'LIKED';
const STATUS_DISLIKED = 'DISLIKED';
const STATUS_NONE = 'NONE';
```

an by default use STATUS_NONE